### ISSUE TITLE
Focus indicator is missing from elements that have submenus/dropdowns

### ISSUE THREAD

**Issue by axamei** [pulibrary/orangelight#1223]
Focus is not indicated for "your account" link and the does not display/activate the dropdown/submenus. Pressing the space bar will activate/display the submenu/dropdown under "your account", but this might not be very clear/intuitive to the user. Same issue exists for the search field options (keyword, title, author, etc.) before the search input.

**Comment by sandbergja** [comment]
This was closed by https://github.com/pulibrary/orangelight/pull/1327 . When I tab to "Your account" or the search field dropdown, I see a nice, blue focus indicator.
